### PR TITLE
Limit of binary actions is too low.

### PR DESCRIPTION
**Pull request by cbickel**
On creating an action, the code is sent to the controller. Afterwards the controller checks, that the code is not too big. If the code itself is sent directly (e.g. uploading a single `.js` file), the limit is 48MB (today).
If an archive is uploaded, it will be encoded with base64. The problem here is, that base64 has an overhead of 1/3. This means, if you have an archive, that has a size of e.g. 45MB, 60MB will be sent to the controller. So the request will be rejected. This is not expected for the user, as a limit of 48MB was proposed to him.

This PR fixes that bug.

## Related issue and scope
- [x] I opened an issue to propose and discuss this change (apache-mailing-list)

## My changes affect the following components
- [ ] API
- [x] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [x] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by rabbah** [comment]
@cbickel will this PR also close https://github.com/apache/incubator-openwhisk/issues/3712?

**Comment by cbickel** [comment]
@rabbah Thanks for the pointer. I wasn't aware of this issue. But I think this PR will close this issue as well.

**Comment by cbickel** [comment]
PG5#479 is 🔵

**Comment by cbickel** [comment]
PG5#1928 🔵

File: tests/src/test/scala/whisk/core/entity/test/SizeTests.scala [hunk 1]
```diff
@@ -93,6 +93,30 @@ class SizeTests extends FlatSpec with Matchers {
...
+  // Division
+  it should "5 Byte / 2 = 2 Byte" in {
+    5.B / 2 should be(2 B)
+  }
+
+  it should "1 MB / 512 = 2 Byte" in {
+    1.MB / 512 should be(2 KB)
+  }
+
+  it should "throw an exception if division is through 0" in {
```
Comment (sven-lange-last) [inline tests/src/test/scala/whisk/core/entity/test/SizeTests.scala]: ... if division is by 0 ...

**Comment by sven-lange-last** [comment]
A more general finding: PR #3209 introduced a configurable limit for parameters in an activation. I guess it makes sense to use the same configuration settings for parameter size and activation parameter size. Can you please check how this could be done?

**Comment by style95** [comment]
Is there any task in-progress regarding this issue?

**Comment by rabbah** [comment]
@cbickel this has rotted - what's the plan?

**Comment by cbickel** [comment]
I'll close the PR for now, due to long inactivity from my side.

**Pull request closed without merging**